importer/tests/test_import.py

```diff
@@ -0,0 +1,56 @@
+"""
+Tests for LORE imports.
+"""
+
+# standard library
+from unittest import TestCase
+from os.path import abspath, dirname, exists, join
+from os import mkdir
+
+# PyPi
+import requests
+
+# local
+from importer.api import import_course_from_file
+
+# TODO: Also test this course:
+# https://github.com/pmitros/edX-Insider/archive/master.zip
+def get_course_zip():
+    """
+    Get the path to the demo course.
+    """
+    url = "https://github.com/edx/edx-demo-course/archive/master.zip"
```

Comment:
Please don't put external dependencies in tests, it makes it really hard to work offline, makes the tests slower, and means that our tests are dependent on github being available.  I would rather add this as a git submodule to the repo, or just straight drop it into the repo.